### PR TITLE
develop: review bookshop schema design

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-bookshop-schema-design.md [hunk 1]
```diff
@@ -4,71 +4,91 @@ title: Bookshop Example Application
...
-### Method 1: Through `tiup demo` command line
+You can import data either by using `tiup demo` or by using the import function of TiDB Cloud.
+
+<SimpleTab>
+<div label="Via `TiUP demo`">
 
-If you use [TiUP](/tiup/tiup-reference.md#tiup-reference) to deploy a TiDB cluster or you can directly connect to your TiDB server, you can quickly generate and import sample data for the Bookshop application with the following commands :
+### Via `tiup demo`
+
+If your TiDB cluster is deployed using [TiUP](/tiup/tiup-reference.md#tiup-reference) or you can connect to your TiDB server, you can quickly generate and import sample data for the Bookshop application by running the following command:
```
Comment (TomShawn) [inline develop/dev-guide-bookshop-schema-design.md:24]:
```suggestion
If your TiDB cluster is deployed using [TiUP](/tiup/tiup-reference.md#tiup-reference) and you can connect to your TiDB server, you can quickly generate and import sample data for the Bookshop application by running the following command:
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 2]
```diff
@@ -4,71 +4,91 @@ title: Bookshop Example Application
...
 
 {{< copyable "shell" >}}
 
 ```shell
 tiup demo bookshop prepare
 ```
 
-This command will connect to the `4000` port on the `127.0.0.1` address by default, use the `root` user to log in with no password, and create a [table structure](#details-about-table) in a database named `bookshop` by default.
+By default, this command connects to port `4000` on address `127.0.0.1`, enables you to log in as the root user without a password, and creates a [table structure](#description-of-the-tables) in a database named `bookshop`.
```
Comment (TomShawn) [inline develop/dev-guide-bookshop-schema-design.md:32]:
```suggestion
By default, this command enables your application to connect to port `4000` on address `127.0.0.1`, enables you to log in as the root user without a password, and creates a [table structure](#description-of-the-tables) in a database named `bookshop`.
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 3]
```diff
@@ -4,71 +4,91 @@ title: Bookshop Example Application
...
+| `--users`   | `10000`  | The number of rows of data to be generated in the `users` table   |
+| `--authors` | `20000`  | The number of rows to be generated in the `authors` table |
+| `--books`   | `20000`  | The number of rows of data to be generated in the `books` table   |
+| `--orders`  | `300000` | The number of rows of data to be generated in the `orders` table  |
+| `--ratings` | `300000` | The number of rows of data to be generated in the `ratings` table |
+
+For example, the following command is executed to generate:
 
-For example, the following command specifies to generate 500,000 lines of basic book information via the `--books` parameter, 100,000 lines of author information via the `--authors` parameter, 1,000,000 lines of rating records via the `--ratings` parameter, and 1,000,000 lines of order records via the `--orders` parameter.
+- 200,000 rows of user information via the `--users` parameter
```
Comment (TomShawn) [inline develop/dev-guide-bookshop-schema-design.md:68]: 位数每隔三位加逗号，主要用在货币数字上：https://developers.google.com/style/numbers

File: develop/dev-guide-bookshop-schema-design.md [hunk 4]
```diff
@@ -135,78 +163,81 @@ WHERE table_schema LIKE 'bookshop';
 6 rows in set (0.03 sec)
 ```
 
-## Details about Table
+</div>
+</SimpleTab>
+
+## Description of the Tables
```
Comment (TomShawn) [inline develop/dev-guide-bookshop-schema-design.md:169]:
```suggestion
## Description of the tables
```

**Review by TomShawn — APPROVED** [review]
minor comments
rest LGTM

File: develop/dev-guide-optimize-sql.md [hunk 5]
```diff
@@ -9,15 +9,15 @@ This section introduce some common reasons for slow SQL statements, and you will
 
 ## Before you begin
 
-You can use [`tiup demo` Import](/develop/dev-guide-bookshop-schema-design.md#method-1-through-tiup-demo-command-line) to prepare data:
+You can use [`tiup demo` Import](/develop/dev-guide-bookshop-schema-design.md#via-tiup-demo) to prepare data:
```
Comment (Mini256) [inline develop/dev-guide-optimize-sql.md:12]:
```suggestion
You can use [`tiup demo` import](/develop/dev-guide-bookshop-schema-design.md#via-tiup-demo) to prepare data:
```

File: develop/dev-guide-hybrid-oltp-and-olap-queries.md [hunk 6]
```diff
@@ -12,15 +12,15 @@ In the [Create Database](/develop/dev-guide-create-table.md#using-htap-capabilit
...
+Before starting, you can import much larger sample data [via the `tiup demo` command](/develop/dev-guide-bookshop-schema-design.md#via-tiup-demo), for example:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 tiup demo bookshop prepare --users=200000 --books=500000 --authors=100000 --ratings=1000000 --orders=1000000 --host 127.0.0.1 --port 4000 --drop-tables
 ```
 
-Or [using the Import function of TiDB Cloud](/develop/dev-guide-bookshop-schema-design.md#method-2-through-the-tidb-cloud-import-function) to import the pre-prepared sample data.
+Or [using the Import function of TiDB Cloud](/develop/dev-guide-bookshop-schema-design.md#via-tidb-cloud-import) to import the pre-prepared sample data.
```
Comment (Mini256) [inline develop/dev-guide-hybrid-oltp-and-olap-queries.md:23]:
```suggestion
Or [using the import function of TiDB Cloud](/develop/dev-guide-bookshop-schema-design.md#via-tidb-cloud-import) to import the pre-prepared sample data.
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 7]
```diff
@@ -135,78 +163,81 @@ WHERE table_schema LIKE 'bookshop';
...
-| id         | bigint(20) | The unique ID of the order                                     |
-| book_id    | bigint(20) | Unique ID of the book (linked to [books](#books-table))        |
+| id         | bigint(20) | Unique ID of an order                                     |
+| book_id    | bigint(20) | Unique ID of a book (linked to [books](#books-table))        |
 | user_id    | bigint(20) | User unique identifier (associated with [users](#users-table)) |
 | quantity   | tinyint(4) | Purchase quantity                                              |
 | ordered_at | datetime   | Purchase time                                                  |
 
-## Database initialization script `dbinit.sql` 
+## Database initialization script `dbinit.sql`
```
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:240]: The new section description needs to be translated:

https://github.com/pingcap/docs-cn/blob/e02b2eab472ac67ab2a9d470dcf2e00447204cc8/develop/dev-guide-bookshop-schema-design.md#%E6%95%B0%E6%8D%AE%E5%BA%93%E5%88%9D%E5%A7%8B%E5%8C%96-dbinitsql-%E8%84%9A%E6%9C%AC

File: develop/dev-guide-bookshop-schema-design.md [hunk 8]
```diff
@@ -78,34 +98,40 @@ On the database details page of TiDB Cloud, you can click the **Import** button
...
+3. Select **TiDB Dumpling** for **Data Format**.
 
-   ![Import Bookshop data in TiDB Cloud](/media/develop/tidb_cloud_import_bookshop_data.png)
+    ![Import Bookshop data in TiDB Cloud](/media/develop/tidb_cloud_import_bookshop_data.png)
 
-4. Enter your database login information.
-5. Click the **Import** button to confirm the import.
-6. Wait for TiDB Cloud to complete the data import.
+4. Enter database login information.
+5. Click the **Import** option to confirm the import.
```
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:121]: I think it's a button, not an option.

File: develop/dev-guide-bookshop-schema-design.md [hunk 9]
```diff
@@ -4,71 +4,91 @@ title: Bookshop Example Application
 
 # Bookshop Example Application
 
-Bookshop is a virtual online bookstore application, you can easily buy books of various categories in Bookshop, and you can also rate on the books you have read.
+Bookshop is a virtual online bookstore application through which you can buy books of various categories and rate the books you have read.
+
+This document describes the following information:
```
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:9]: You can refer to new changes on the chinese document, our website already has a page navigation feature, no need to add redundant outlines.

https://github.com/pingcap/docs-cn/blob/e02b2eab472ac67ab2a9d470dcf2e00447204cc8/develop/dev-guide-bookshop-schema-design.md#bookshop-%E5%BA%94%E7%94%A8

File: develop/dev-guide-bookshop-schema-design.md (hunk 8 above)
Comment (shichun-0415) [reply develop/dev-guide-bookshop-schema-design.md:121]: > I think it's a button, not an option.
Please see the word list of [Google developer documentation style guide](https://developers.google.cn/style/word-list#letter-b)
image

File: develop/dev-guide-bookshop-schema-design.md (hunk 1 above)
Comment (shichun-0415) [reply develop/dev-guide-bookshop-schema-design.md:24]: @Mini256 Could you help confirm this modification. In the Chinese version, the word is "或者”, indicating either situation applies.

File: develop/dev-guide-bookshop-schema-design.md (hunk 8 above)
Comment (Mini256) [reply develop/dev-guide-bookshop-schema-design.md:121]: Let us look at this guide in conjunction with practical examples:

https://cloud.google.com/s/results?q=option
https://cloud.google.com/s/results?q=button

I think this document ignores the button description of the PC side, and I believe **Import** is an on-screen button.

File: develop/dev-guide-bookshop-schema-design.md (hunk 1 above)
Comment (Mini256) [reply develop/dev-guide-bookshop-schema-design.md:24]: use `or`, this means that if you use TIDB Cloud to start the TIDB cluster, you can use this command as long as you can connect.

File: develop/dev-guide-bookshop-schema-design.md [hunk 10]
```diff
@@ -4,71 +4,87 @@ title: Bookshop Example Application
 
 # Bookshop Example Application
 
-Bookshop is a virtual online bookstore application, you can easily buy books of various categories in Bookshop, and you can also rate on the books you have read.
+Bookshop is a virtual online bookstore application through which you can buy books of various categories and rate the books you have read.
 
-## Import data
+To make your reading on the application developer guide more smoothly, we present the SQL statements based on the [table structures](#description-of-the-tables) and data of the Bookshop application. This document focuses on the methods of importing the table structures and data as well as the definitions of the table structures.
```
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:9]:
```suggestion
To make your reading on the application developer guide more smoothly, we present the example SQL statements based on the [table structures](#description-of-the-tables) and data of the Bookshop application. This document focuses on the methods of importing the table structures and data as well as the definitions of the table structures.
```

File: develop/dev-guide-bookshop-schema-design.md [hunk 11]
```diff
@@ -4,71 +4,87 @@ title: Bookshop Example Application
...
+If your TiDB cluster is deployed using [TiUP](/tiup/tiup-reference.md#tiup-reference) or you can connect to your TiDB server, you can quickly generate and import sample data for the Bookshop application by running the following command:
 
 {{< copyable "shell" >}}
 
 ```shell
 tiup demo bookshop prepare
 ```
 
-This command will connect to the `4000` port on the `127.0.0.1` address by default, use the `root` user to log in with no password, and create a [table structure](#details-about-table) in a database named `bookshop` by default.
+By default, this command enables your application to connect to port `4000` on address `127.0.0.1`, enables you to log in as the root user without a password, and creates a [table structure](#description-of-the-tables) in the database named `bookshop`.
```
Comment (shichun-0415) [inline develop/dev-guide-bookshop-schema-design.md:28]:
```suggestion
By default, this command enables your application to connect to port `4000` on address `127.0.0.1`, enables you to log in as the `root` user without a password, and creates a [table structure](#description-of-the-tables) in the database named `bookshop`.
```

**Review by Mini256 — APPROVED** [review]
Thank you for your suggestion and review. 👍

File: develop/dev-guide-hybrid-oltp-and-olap-queries.md [hunk 12]
```diff
@@ -12,15 +12,15 @@ In the [Create Database](/develop/dev-guide-create-table.md#using-htap-capabilit
 
 ## Data preparation
 
-Before starting, you can import much larger sample data [via the `tiup demo` command](/develop/dev-guide-bookshop-schema-design.md#method-1-through-tiup-demo-command-line), for example:
+Before starting, you can import much larger sample data [via the `tiup demo` command](/develop/dev-guide-bookshop-schema-design.md#via-tiup-demo), for example:
```
Comment (shichun-0415) [inline develop/dev-guide-hybrid-oltp-and-olap-queries.md:15]:
```suggestion
You can use [`tiup demo` import](/develop/dev-guide-bookshop-schema-design.md#via-tiup-demo) to prepare data:
```

File: develop/dev-guide-optimize-sql.md [hunk 13]
```diff
@@ -9,15 +9,15 @@ This document introduces some common reasons for slow SQL statements and techniq
...
+You can use [`tiup demo` import](/develop/dev-guide-bookshop-schema-design.md#via-tiup-demo) to prepare data:
 
 {{< copyable "shell-regular" >}}
 
 ```shell
 tiup demo bookshop prepare --host 127.0.0.1 --port 4000 --books 1000000
 ```
 
-Or [using the Import function of TiDB Cloud](/develop/dev-guide-bookshop-schema-design.md#method-2-through-the-tidb-cloud-import-function).
+Or [using the Import function of TiDB Cloud](/develop/dev-guide-bookshop-schema-design.md#via-tidb-cloud-import) to import the pre-prepared sample data.
```
Comment (shichun-0415) [inline develop/dev-guide-optimize-sql.md:20]:
```suggestion
Or [using the Import feature of TiDB Cloud](/develop/dev-guide-bookshop-schema-design.md#via-tidb-cloud-import) to import the pre-prepared sample data.
```

**Pull request merged**